### ISSUE TITLE
Kerbal's randomly KIA after recovering vessel.

### ISSUE THREAD

**Issue by nexitem** [LunaMultiplayer/LunaMultiplayer#136]
Issue:

I know that there is a shit ton of mods installed, but I am quite sure this happens because of USI Life Support, cause when my friend takes a pilot with him and he recovers the vessel they appear as "KIA" again. I think I will test this tomorrow with just USI installed to see if it is still happening :-)

------------------------------------------------------------------------------

OS: Windows 10

------------------------------------------------------------------------------

KSPVersion: 1.4.3

------------------------------------------------------------------------------

LMP version: Build 0.8.42.406

------------------------------------------------------------------------------

MODS, Universe & Configs:
https://trior.net/u/nexitem/public_html/GameData.rar
https://trior.net/u/nexitem/public_html/Universe.rar 
https://trior.net/u/nexitem/public_html/Config.rar
https://trior.net/u/nexitem/public_html/LMPModControl.xml

------------------------------------------------------------------------------

Expected behaviour: Astronauts should not die or get lost when players recover vessels.

------------------------------------------------------------------------------

Current behaviour: Sometimes Astronauts die randomly or get unavailable

------------------------------------------------------------------------------

**Steps to reproduce:**
Send a rocket with 1 pilot on a mission, have USI Life Support set on "KIA" for astronauts. Finish mission, recover the vessel and the kerbal won't be back but "KIA" instead.

**Comment by nexitem** [comment]
Ok after visiting the discord and reading into past problems other users have I realized that the developers of LMP mentioned that mods that have food or life support might not be supported or cause problems. As I suspected this is probably caused by USI Life Support.

So yeah... :) can be closed I guess? Or keep it open if you guys think there might be a way to make these things work in the future, But there are other things that have priority.
